packages/openapi/src/selection/duplicate.ts

```diff
@@ -0,0 +1,51 @@
+import { axios } from '../axios';
+import { registerRoute, urlBuilder } from '../utils';
+import { z } from '../zod';
+import type { IRangesRo } from './range';
+
+export const DUPLICATE_URL = '/table/{tableId}/selection/duplicate';
+
+export const duplicateVoSchema = z.object({
+  ids: z.array(z.string()),
+});
+
+export type IDuplicateVo = z.infer<typeof duplicateVoSchema>;
+
+export const duplicateRoute = registerRoute({
+  method: 'get',
+  path: DUPLICATE_URL,
+  description: 'Duplicate the selected data',
+  request: {
+    params: z.object({
+      tableId: z.string(),
+    }),
```

Comment:
The "order" (which can be found in the "record/create") parameter should be included, and the copied records will be inserted below the target records.